portfolio/src/main/webapp/style.css

```diff
@@ -21,44 +24,37 @@ body {
 }
 
 h1 {
-  font-family: "Montserrat";
-  font-weight: 300;
-  font-size: 3em;
+  font-family: "Roboto";
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#CSS_Quotation_Marks): Use single (`''`) rather than double (`""`) quotation marks for attribute selectors and property values.

Here and elsewhere.